10.11/root/usr/share/container-scripts/mysql/README.md

```diff
@@ -0,0 +1,369 @@
+       To enable query logging set this to `1`
+
+
+You can also set the following mount points by passing the `-v /host:/container` flag to Docker.
+
+**`/var/lib/mysql/data`**  
+       MySQL data directory
+
+
+**Notice: When mouting a directory from the host into the container, ensure that the mounted
+directory has the appropriate permissions and that the owner and group of the directory
+matches the user UID or name which is running inside the container.**
+
+
+MariaDB auto-tuning
+-------------------
+
+When the MySQL image is run with the `--memory` parameter set and you didn't
+specify value for some parameters, their values will be automatically
+calculated based on the available memory.
+
+**`MYSQL_KEY_BUFFER_SIZE (default: 10%)`**  
+       `key_buffer_size`
+
+**`MYSQL_READ_BUFFER_SIZE (default: 5%)`**  
+       `read_buffer_size`
+
+**`MYSQL_INNODB_BUFFER_POOL_SIZE (default: 50%)`**  
+       `innodb_buffer_pool_size`
+
+**`MYSQL_INNODB_LOG_FILE_SIZE (default: 15%)`**  
+       `innodb_log_file_size`
+
+**`MYSQL_INNODB_LOG_BUFFER_SIZE (default: 15%)`**  
+       `innodb_log_buffer_size`
+
+
+
+MySQL root user
+---------------------------------
+The root user has no password set by default, only allowing local connections.
+You can set it by setting the `MYSQL_ROOT_PASSWORD` environment variable. This
+will allow you to login to the root account remotely. Local connections will
+still not require a password.
+
+To disable remote root access, simply unset `MYSQL_ROOT_PASSWORD` and restart
+the container.
+
+
+Changing passwords
+------------------
+
+Since passwords are part of the image configuration, the only supported method
+to change passwords for the database user (`MYSQL_USER`) and root user is by
+changing the environment variables `MYSQL_PASSWORD` and `MYSQL_ROOT_PASSWORD`,
+respectively.
+
+Changing database passwords through SQL statements or any way other than through
+the environment variables aforementioned will cause a mismatch between the
+values stored in the variables and the actual passwords. Whenever a database
+container starts it will reset the passwords to the values stored in the
+environment variables.
+
+
+Default my.cnf file
+-------------------
+With environment variables we are able to customize a lot of different parameters
+or configurations for the mysql bootstrap configurations. If you'd prefer to use
+your own configuration file, you can override the `MYSQL_DEFAULTS_FILE` env
+variable with the full path of the file you wish to use. For example, the default
+location is `/etc/my.cnf` but you can change it to `/etc/mysql/my.cnf` by setting
+ `MYSQL_DEFAULTS_FILE=/etc/mysql/my.cnf`
+
+
+Extending image
+---------------
+This image can be extended in Openshift using the `Source` build strategy or via the standalone
+[source-to-image](https://github.com/openshift/source-to-image) application (where available).
+For this, we will assume that you are using the `rhscl/mariadb-1011-rhel7` image,
```

Comment:
Also rhel7 will not get mariadb 10.11.

Reply:
Fixed by https://github.com/sclorg/mariadb-container/pull/223/commits/0d6ea1b236e5a3f0a9c0d28e300ecc4feb49c66e